scripts/liminal

```diff
@@ -105,17 +108,15 @@ def deploy_liminal_apps(path, clean):
         shutil.copyfile(config_file, target_yml_name)
 
 def is_airflow_running():
-        result = subprocess.check_output([f'docker-compose -f "{environment.get_liminal_home()}/docker-compose.yml" ps'], shell=True, universal_newlines=True)
+        result = docker_compose_command('check_output', 'ps', [])
         return "liminal" in result
 
-@cli.command("stop", short_help="stops the docker compose.")
+@cli.command("stop", short_help="stops the docker compose")
 def stop():
-
     if docker_is_running() and is_airflow_running():
         # initialize liminal home by default
         environment.get_liminal_home()
-        result = subprocess.call([f'docker-compose -f "{environment.get_liminal_home()}/docker-compose.yml" down'],
-                                 env=os.environ, shell=True)
+        docker_compose_command('call', 'down', [])
```

Comment:
did you check if it kills all the containers after down?

Reply:
Yes